test/specs/scenarios/face.js

```diff
@@ -340,7 +340,7 @@ export const faceScenarios = (lang) => {
       faceVideoIntro.verifyUIElementsOnTheFaceVideoIntroScreen(copy)
       faceVideoIntro.clickOnContinueButton()
       camera.enableCameraButton().click()
-      camera.verifyVideoTitle(copy)
+      //camera.verifyVideoTitle(copy) - copy difference
```

Comment:
What was the copy difference here?

Reply:
Oh, I just ran it uncommented, it looks ok now, there might have been a copy issue when I first created the branch.